core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -166,6 +172,7 @@ public boolean isDeveloperMode() {
 
     /**
      * Sets value indicating whether this channel is in developer mode.
+     *
```

Comment:
empty line. did you mean?

Reply:
This should be by mistake I would remove. Thanks